new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMark.java

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes;
+
+import io.kubernetes.client.openapi.ApiException;
+import oracle.weblogic.kubernetes.annotations.IntegrationTest;
+import oracle.weblogic.kubernetes.extensions.LoggedTest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static oracle.weblogic.kubernetes.actions.TestActions.createSecret;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.catchThrowable;
+
+@DisplayName("Simple validation of basic domain functions")
+@IntegrationTest
+class ItMark implements LoggedTest {
+
+  @Test
+  @DisplayName("Test something")
+  public void testSomeStuff() {
+    int x = 1 + 5;
+    logger.info("log message");
+
+    assertThat(true).as("Test that true is true")
+        .withFailMessage("OMG! True was not true")
+        .isEqualTo(true);
+  }
+
+  @Test
+  @DisplayName("Test something else")
+  public void testThereWasNoException() {
+    // this test will fail because it does throw an exception!
+
+    assertThatCode(() ->
+        createSecret("bob", // name
```

Comment:
if createSecret returns a string, how to capture the return value?

Reply:
I am looking to write assertions for createUniqueNamespace() call, I need to capture the return value.

Reply:
I pushed an update that does that.  It is not quite so "pretty" - but it does work, and it is easy enough to read it and understand what it means

Reply:
currently I have a mix of Junit assertions and assertJ in ItSimpleOperatorValidation.java, thought I would use assertJ for all, but this looks like adding more code if I want to replace assertDoesNotThrow calls.

Reply:
that's ok - whatever is the most readable is fine

Reply:
Ok. I modified the code, I have like this 
```
    String opNamespace = null;
    try {
      opNamespace = createUniqueNamespace();
    } catch (Exception e) {
      assertThat(e)
          .as("Test that createUniqueNamespace does not throw an exception")
          .withFailMessage("createUniqueNamespace() threw an unexpected exception")
          .isNotInstanceOf(Exception.class);
    }
```
later I use this variable to delete the namespace using lambda - 
```
      assertThatCode(
          () -> deleteNamespace(opNamespace))
          .as("Test that deleteNamespace doesn not throw an exception")
          .withFailMessage("deleteNamespace() threw an exception")
          .doesNotThrowAnyException();
```
 It gives error 
```
local variables referenced from a lambda expression must be final or effectively final
```
so, I can't use lambda here also as I have non-final variable but use try catch block.

Reply:
don't you need to name the variable before the `-?` to pass it in?